base/util.jl

```diff
@@ -271,15 +271,29 @@ function securezero! end
 unsafe_securezero!(p::Ptr{Cvoid}, len::Integer=1) = Ptr{Cvoid}(unsafe_securezero!(Ptr{UInt8}(p), len))
 
 """
-    Base.getpass(message::AbstractString) -> Base.SecretBuffer
+    Base.getpass(message::AbstractString; with_suffix::Bool=true) -> Base.SecretBuffer
```

Comment:
I wonder if it would be a cleaner API to pass `prompt_suffix::AbstractString = ": "`, allowing the caller to change the suffix to whatever they want, and pass `prompt_suffix=""` to omit the suffix.

Reply:
Yeah I mentioned that in the first post, basically I can't think of a good usecase for the user putting the suffix in a separate argument instead of the message itself so I went with this API.